daemon/graphdriver/quota/projectquota.go

```diff
@@ -194,6 +247,49 @@ func setProjectQuota(backingFsBlockDev string, projectID uint32, quota Quota) er
 	return nil
 }
 
+// Ext4Quota is a struct implements quota operations
+type Ext4Quota struct {
+}
+
+// SetProjectQuota - set the quota for project id on ext4 block device
+func (q *Ext4Quota) SetProjectQuota(backingFsBlockDev string, projectID uint32, quota Quota) error {
```

Comment:
Ok, but for future code sake let's use Q_X.. for xfsQuota and Q_... For VfsQuota and try to avoid checking for ext4 explicitly when choosing VfsQuota. Thanks